addons/io_scene_gltf2/__init__.py

```diff
@@ -336,6 +332,17 @@ def invoke(self, context, event):
                 self.report({"ERROR"}, "Loading export settings failed. Removed corrupted settings")
                 del context.scene[self.scene_key]
 
+        self.user_extensions = {}
+        import sys
+        for addon_name in bpy.context.preferences.addons.keys():
+            try:
+                # Every user-extension would have a top-level class called Gltf2UserExt
```

Comment:
The comment says `Gltf2UserExt` but the code uses `glTF2UserExtension`. (You can just throw away my comment here if you want.)